frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/create-local-volume-set.tsx

```diff
@@ -0,0 +1,236 @@
+            ]}
+          />
+        </div>
+        <h1 className="co-create-operand__header-text">{`Create ${modelName}`}</h1>
+        <p className="help-block">
+          A {modelName} allows you to filter a set of storage volumes, group them and create a
+          dedicated storage class to consume storage for them.
+        </p>
+      </div>
+      <Form noValidate={false} className="co-m-pane__body co-m-pane__form" onSubmit={onSubmit}>
+        <FormGroup label="Volume Set Name" isRequired fieldId="create-lvs--volume-set-name">
+          <TextInput
+            type={TextInputTypes.text}
+            id="create-lvs--volume-set-name"
+            value={volumeSetName}
+            onChange={setVolumeSetName}
+            isRequired
+          />
+        </FormGroup>
+        <FormGroup label="Storage Class Name" fieldId="create-lvs--storage-class-name">
+          <TextInput
+            type={TextInputTypes.text}
+            id="create-lvs--storage-class-name"
+            value={storageClassName}
+            onChange={setStorageClassName}
+            isRequired
+          />
+        </FormGroup>
+        <Text component={TextVariants.h3} className="lso-create-lvs__filter-volumes-text--margin">
+          Filter Volumes
+        </Text>
+        <FormGroup label="Node Selector" fieldId="create-lvs--radio-group-node-selector">
+          <div id="create-lvs--radio-group-node-selector">
+            <Radio
+              label="All nodes"
+              name="nodes-selection"
+              id="create-lvs--radio-all-nodes"
+              className="lso-create-lvs__all-nodes-radio--padding"
+              value="allNodes"
+              onChange={toggleShowNodesList}
+              description="Selecting all nodes will search for available volume storage on all nodes."
+              defaultChecked
+            />
+            <Radio
+              label="Select nodes"
+              name="nodes-selection"
+              id="create-lvs--radio-select-nodes"
+              value="selectedNodes"
+              onChange={toggleShowNodesList}
+              description="Selecting nodes allow you to limit the search for available volumes to specific nodes."
+            />
+          </div>
+        </FormGroup>
+        {showNodesList && (
+          <ListPage
+            customData={{ rows, setRows, allSelected, setAllSelected }}
+            showTitle={false}
+            kind={NodeModel.kind}
+            ListComponent={NodesSelectionList}
+          />
+        )}
+        <FormGroup label="Volume Type" fieldId="create-lvs--volume-type-dropdown">
+          <Dropdown
+            id="create-lvs--volume-type-dropdown"
+            dropDownClassName="dropdown--full-width"
+            items={volumeTypeDropdownItems}
+            title={volumeTypeDropdownItems[volumeType]}
+            selectedKey={volumeType}
+            onChange={setVolumeType}
+          />
+        </FormGroup>
+        <Expandable toggleText="Advanced">
```

Comment:
Please put a data-test-id on the toggler.

Reply:
Added